COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md

```diff
@@ -4299,3 +4299,210 @@ Previous lab programs made use of a date/time intrinsic function.  The date/time
 **Lab Hints**
 
 Refer to CBL0011 line 120 for the proper formatting of the function-name causing   the compile error.
+
+\newpage
+# ABEND handling
```

Comment:
Please review how other chapters are set up with a brief introduction followed by a table of contents for the chapter.

Reply:
Fixed in d52e3ff.